app/models/person.rb

```diff
@@ -41,6 +43,7 @@ class Person < ActiveRecord::Base
   include Notable
   include MediaAttachable
 
+  VALID_DIVISION_ROLES = [:member, :admin]
```

Comment:
You can use `%i(member admin)` in cases like this. Doesn't make much diff here, but sometimes nice.